pkg/crd/crd.go

```diff
@@ -306,6 +306,15 @@ func (c *Client) RemoveAssignedIdentity(assignedIdentity *aadpodid.AzureAssigned
 	err := c.rest.Delete().Namespace(assignedIdentity.Namespace).Resource("azureassignedidentities").Name(assignedIdentity.Name).Do().Error()
 	glog.V(5).Infof("Deletion %s took: %v", assignedIdentity.Name, time.Since(begin))
 	stats.Update(stats.AssignedIDDel, time.Since(begin))
+
+	metrics.NewReporterAndReport(
```

Comment:
Can we add the reporter to client struct and use that for all the reports? That way we can initialize it when we create the client the first time.

Reply:
moved.